docker-jans-monolith/README.md

```diff
@@ -32,35 +42,38 @@ The following environment variables are supported by the container:
 | `MYSQL_HOST`            | MySQL host.                                      | `mysql` which is the docker compose service name |
 
 
-## Pre-requisites
+## How to run
 
-- [Docker](https://docs.docker.com/install). Docker compose should be installed by default with Docker.
+Download the compose file 
 
-## How to run
+```bash
+
+wget https://raw.githubusercontent.com/JanssenProject/jans/main/docker-jans-monolith/jans-mysql-compose.yml 
+```
+
+This docker compose file run two containers, the janssen monolith container and mysql container.
 
 ```bash
 docker compose -f jans-mysql-compose.yml up -d
 ```
 
-## Clean up
+To see the containers running
 
-Remove setup and volumes
+```bash
 
-```
-docker compose -f jans-mysql-compose.yml down && rm -rf jans-*
+docker ps
```

Comment:
Since containers are deployed using `docker compose`, all related commands should be run with `docker compose` instead of native `docker`. For example, running `docker ps` will list all containers including the ones that not deployed via `docker compose`.